.lintstagedrc

```diff
@@ -1,5 +1,5 @@
 
```

Comment:
Shouldn't this leading new line also go?

```suggestion
```

Renaming the file to `.lintstagedrc.json` might help actually. 🙂

Reply:
It doesn't seem like Prettier cared about that, but I'm down to remove it -- ultimately, Prettier will take care of the formatting now since the most recent update, so we won't have to worry about it going forward!